test/logintest.js

```diff
@@ -0,0 +1,143 @@
+const request = require('supertest');
+const app = require('../api/server').app;
+const assert = require('assert');
+const db = require('../api/models');
+const config = require('../api/config');
+const clone = require('clone');
+const fs = require('fs');
+
+const errorMessage = "Error: This is a top-10 common password - Add another word or two. Uncommon words are better.. Please choose a stronger password and try again.";
+describe('Signup API', () => {
+  before((done) => {
+    // Connect to test database
+    db.init(() => {
+      // Clear test database
+      dbConnection = db.init(config.mongodb_test, done);
```

Comment:
it would be preferable to use `config.mongodb` and evaluate the NODE_ENV var to check when on test mode, and if it is, use the mongo test db.

e.g. on config.js:
```js
mongodb: NODE_ENV === 'test' ? 'mongodb://...' : 'mongodb://...' 
```
unfortunately config are in JS code, which is not good for infrastructure support. we may move to .env files in the future